src/loader/cmt.ml

```diff
@@ -392,8 +416,19 @@ and unwrap_module_expr_desc = function
 and read_module_binding env parent mb =
   let open Module in
   let open Odoc_model.Names in
-  let name = parenthesise (Ident.name mb.mb_id) in
-  let id = `Module(parent, ModuleName.of_string name) in
+  let id =
+#if OCAML_MAJOR = 4 && OCAML_MINOR >= 10
+      match mb.mb_id with
+      | Some id ->
+        let name = parenthesise (Ident.name id) in
+        `Module(parent, ModuleName.of_string name)
+      | None ->
+        `Module(parent, ModuleName.of_string "_")
+#else
```

Comment:
As with `read_module_declaration`, `read_module_binding` should not produce an item when there is no identifier.

Reply:
Done.